aries-site/src/pages/foundation/accessibility.mdx

```diff
@@ -114,3 +113,25 @@ As a designer or developer, we are morally responsible for creating accessible p
 - [WebAIM Introduction to Web Accessibility](https://webaim.org/intro/): An introduction on how accessibility improves the Web for all users.
 - [The A11y Project Checklist](https://www.a11yproject.com/checklist/): A checklist using WCAG 2.1 as a reference for developers and designers to ensure their product's accessibility.
 - [U.S. Web Design System Principles](https://designsystem.digital.gov/design-principles/#embrace-accessibility): These principles and key guidelines outline how you should approach accessibility from a design system perspective.
+
+## Accessibility support at different levels
```

Comment:
I think this section should go directly below the "Developing for accessibility" section

Reply:
ah okay my bad thought it was at the end